api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/fields/format.md

```diff
@@ -162,6 +162,207 @@ See the [format](/api-reference/50%20Common/Object%20Structures/format '/Documen
 
 ---
 
+Date formats require additional configuration. The **PivotGridDataSource** groups date field values by date component: Year-Quarter-Month. For this, the values are converted from dates to numbers, but numbers cannot be formatted using date formats. To apply a date format in this case, you need to override this grouping.
```

Comment:
I might be out of context, but what is 'date component'? Is this a legit term?

Can we also replace using with smth like 'you cannot apply date formats to numeric values'?

Reply:
> what is 'date component'?

Year, quarter, month.

> Can we also replace 'numbers cannot be formatted using date formats' with smth like 'you cannot apply date formats to numeric values'?

This kinda breaks the flow in the sentence. I would prefer to keep it as is.